# **📊 1: How many armor pieces are available?**

In [0]:
%sql
SELECT COUNT(*) AS total_armors FROM eldenringcatalog.gold.dim_armors;

total_armors 723

In [0]:
%sql
SELECT COUNT(DISTINCT armor_name) AS unique_names FROM eldenringcatalog.gold.dim_armors;

unique_names 723

In [0]:
%sql
SELECT armor_type, COUNT(*) as count FROM eldenringcatalog.gold.dim_armors GROUP BY armor_type;

armor_type count helm 212 chest armor 251 gauntlets 123 leg armor 137

**723 items distributed across four main categories. Chest armor is the most represented type with 251 items, followed by helms with 212 items. Leg armor includes 137 items, while gauntlets make up the smallest category with 123 items. Overall, the distribution shows a balanced selection of armor pieces, with a slightly higher focus on core protective gear such as chest armor and helms.**

# **📊 2: Armor types distribution**

In [0]:
%sql
SELECT 
    armor_type,
    COUNT(*) AS piece_count,
    ROUND(AVG(weight), 2) AS avg_weight
FROM eldenringcatalog.gold.dim_armors
GROUP BY armor_type
ORDER BY piece_count DESC;

armor_type piece_count avg_weight chest armor 251 8.91 helm 212 4.3 leg armor 137 5.74 gauntlets 123 3.25

**The armor weights follow a clear pattern based on coverage. Chest armor is the heaviest, averaging 8.91, making it the main source of protection. Leg armor comes next with an average weight of 5.74, followed by helms at 4.3. Gauntlets are the lightest at 3.25, reflecting their smaller size and limited coverage. Overall, weight decreases consistently from larger armor pieces to smaller ones.**

# **📊 3: Load armor stats and analyze physical negation**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    negation_physical,
    negation_strike,
    negation_slash,
    negation_pierce,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY negation_physical DESC
LIMIT 30;

armor_name armor_type negation_physical negation_strike negation_slash negation_pierce weight Bull-Goat Armor chest armor 20.4 20.2 18.3 18.3 26.5 Verdigris Armor chest armor 20.4 17.7 19.7 18.7 25.9 Fire Prelate Armor chest armor 19.2 17.0 18.3 18.3 24.7 Armor of Solitude chest armor 18.7 17.7 18.3 17.5 24.0 Tree Sentinel Armor chest armor 18.7 15.0 18.7 17.5 18.9 Fire Prelate Armor (Altered) chest armor 18.7 16.3 17.5 17.5 23.6 Banished Knight Armor chest armor 18.7 15.0 19.2 17.5 17.5 Radahn's Lion Armor chest armor 18.7 15.0 18.3 17.5 17.5 Veteran's Armor chest armor 18.7 17.0 18.7 17.5 18.9 Radahn's Lion Armor (Altered) chest armor 18.5 14.6 17.7 17.0 16.4 Tree Sentinel Armor (Altered) chest armor 18.5 14.8 18.5 17.3 18.1 Omen Armor chest armor 18.3 15.0 18.7 18.7 23.1 Banished Knight Armor (Altered) chest armor 18.3 14.4 18.7 16.8 16.5 Armor of Solitude (Altered) chest armor 18.3 17.0 17.5 16.8 23.2 Veteran's Armor (Altered) chest armor 18.1 16.8 18.5 17.3 17.2 Crucible Tree Armor chest armor 17.7 13.9 17.5 15.8 15.5 Beast Champion Armor chest armor 17.5 16.3 18.7 18.3 17.5 Lionel's Armor chest armor 17.5 16.3 19.2 19.7 21.2 Crucible Axe Armor chest armor 17.5 13.9 16.8 16.8 15.5 Beast Champion Armor (Altered) chest armor 17.2 15.9 18.5 18.2 16.4 Lionel's Armor (Altered) chest armor 17.1 15.9 18.8 19.3 20.2 Crucible Tree Armor (Altered) chest armor 17.0 13.4 16.0 15.6 14.9 Malformed Dragon Armor chest armor 16.8 15.0 17.5 16.8 16.0 Gaius's Armor chest armor 16.8 15.7 19.7 19.2 20.1 Young Lion's Armor chest armor 16.8 14.4 16.8 16.8 15.0 Crucible Axe Armor (Altered) chest armor 16.8 13.4 16.0 16.0 14.9 Scaled Armor chest armor 16.0 13.9 16.8 16.0 16.0 Royal Knight Armor chest armor 16.0 14.4 17.5 16.8 15.5 Royal Knight Armor (Altered) chest armor 15.8 14.2 17.3 16.6 15.0 Young Lion's Armor (Altered) chest armor 15.3 13.4 15.3 15.3 13.7

**The top 30 armor pieces ranked by physical negation are dominated entirely by heavy chest armor, with values ranging from 20.4 down to 15.3. The Bull-Goat Armor and Verdigris Armor lead with the highest physical negation at 20.4, followed closely by other extremely heavy sets like Fire Prelate, Tree Sentinel, Banished Knight, and Radahn's Lion variants. Across all entries, high physical defense consistently pairs with strong strike, slash, and pierce negation, confirming that the tank-oriented, ultra-heavy chest pieces offer the best overall physical protection in the dataset.**

# **📊 4: Best physical protection per weight ratio**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    negation_physical,
    weight,
    ROUND(negation_physical / weight, 2) AS protection_per_weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY protection_per_weight DESC
LIMIT 30;

armor_name armor_type negation_physical weight protection_per_weight Circlet of Light helm 2.8 1.0 2.8 Ascetic's Loincloth chest armor 6.1 3.2 1.91 Braided Cord Robe chest armor 6.3 3.4 1.85 Goldmask's Rags chest armor 4.2 2.4 1.75 Aristocrat Garb (Altered) chest armor 7.1 4.1 1.73 Old Aristocrat Shoes leg armor 3.4 2.0 1.7 Ascetic's Ankle Guards leg armor 3.4 2.0 1.7 Cloth Garb chest armor 5.3 3.2 1.66 Aristocrat Garb chest armor 7.8 4.9 1.59 Aristocrat Headband helm 1.9 1.2 1.58 Soiled Loincloth leg armor 2.3 1.5 1.53 Gravebird Anklets leg armor 3.0 2.0 1.5 Cloth Trousers leg armor 3.0 2.0 1.5 Traveler's Clothes chest armor 6.1 4.1 1.49 Messmer Soldier Armor (Altered) chest armor 12.1 8.1 1.49 Godskin Apostle Robe chest armor 6.1 4.1 1.49 Lazuli Robe chest armor 6.1 4.1 1.49 Aristocrat Boots leg armor 4.3 2.9 1.48 Malenia's Armor (Altered) chest armor 10.0 6.8 1.47 Fire Knight Armor (Altered) chest armor 11.9 8.4 1.42 Raging Wolf Armor (Altered) chest armor 12.8 9.0 1.42 Malenia's Armor chest armor 10.9 7.7 1.42 Blue Silver Mail Armor (Altered) chest armor 12.1 8.5 1.42 Common Soldier Cloth Armor chest armor 12.4 8.8 1.41 Tree Surcoat chest armor 12.9 9.2 1.4 Eye Surcoat chest armor 12.9 9.2 1.4 Page Garb (Altered) chest armor 6.3 4.5 1.4 Leather Headband helm 1.4 1.0 1.4 Freyja's Armor (Altered) chest armor 12.9 9.3 1.39 Freyja's Armor chest armor 13.5 9.7 1.39

**The best protection-per-weight ratios come almost entirely from lightweight gear, with the Circlet of Light outperforming all armor pieces at an exceptional 2.8 protection per weight. Very light chest pieces like the Ascetic’s Loincloth, Braided Cord Robe, and Goldmask’s Rags also rank highly, offering efficient physical defense despite their low overall stats. Several light leg pieces and helms appear as well, all staying under 3–5 weight. As the list progresses, slightly heavier but still efficient altered chest armors—such as Malenia’s Armor (Altered), Fire Knight Armor (Altered), and Raging Wolf Armor (Altered)—enter with solid ratios around 1.4. Overall, the data clearly shows that light armor provides the best physical protection relative to its weight, making it ideal for mobility-focused builds.**

# **📊 6: Poise per weight ratio**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    resistance_poise,
    weight,
    ROUND(resistance_poise / weight, 2) AS poise_per_weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY poise_per_weight DESC
LIMIT 30;

armor_name armor_type resistance_poise weight poise_per_weight Circlet of Light helm 5.0 1.0 5.0 Ascetic's Loincloth chest armor 10.0 3.2 3.13 Ascetic's Ankle Guards leg armor 6.0 2.0 3.0 Gravebird Anklets leg armor 5.0 2.0 2.5 Aristocrat Garb chest armor 12.0 4.9 2.45 Ansbach's Attire (Altered) chest armor 10.0 4.1 2.44 Battlemage Robe chest armor 10.0 4.1 2.44 Aristocrat Garb (Altered) chest armor 10.0 4.1 2.44 Aristocrat Boots leg armor 7.0 2.9 2.41 Battlemage Legwraps leg armor 6.0 2.5 2.4 Fingerprint Armor (Altered) chest armor 24.0 10.0 2.4 Godskin Noble Trousers leg armor 6.0 2.5 2.4 High Priest Undergarments leg armor 6.0 2.5 2.4 High Priest Robe chest armor 10.0 4.2 2.38 Ansbach's Attire chest armor 12.0 5.1 2.35 Finger Robe chest armor 12.0 5.1 2.35 Mausoleum Knight Armor (Altered) chest armor 25.0 10.8 2.31 Eye Surcoat chest armor 21.0 9.2 2.28 Tree Surcoat chest armor 21.0 9.2 2.28 Fingerprint Armor chest armor 24.0 10.6 2.26 Knight Armor chest armor 24.0 10.6 2.26 Ansbach's Boots leg armor 7.0 3.1 2.26 All-Knowing Armor (Altered) chest armor 21.0 9.3 2.26 Mausoleum Surcoat chest armor 24.0 10.6 2.26 Omenkiller Robe chest armor 16.0 7.1 2.25 Redmane Knight Armor (Altered) chest armor 24.0 10.8 2.22 Messmer Soldier Armor (Altered) chest armor 18.0 8.1 2.22 Godrick Knight Armor (Altered) chest armor 24.0 10.8 2.22 Gelmir Knight Armor (Altered) chest armor 24.0 10.8 2.22 Leyndell Knight Armor (Altered) chest armor 24.0 10.8 2.22

**Poise efficiency is led by extremely lightweight pieces, with the Circlet of Light standing out at a massive 5 poise per weight, far higher than any other armor. Other top performers include the Ascetic’s Ankle Guards and Ascetic’s Loincloth, both offering unusually high poise for their very low weight. Beyond these outliers, many mid-weight chest armors—such as Fingerprint Armor, Mausoleum Knight Armor (Altered), and various Knight or Surcoat variants—deliver strong poise-per-weight ratios around 2.2–2.4, balancing sturdiness with manageable weight. Overall, the data shows that while heavy armor provides high total poise, the most weight-efficient poise comes from specific light or altered pieces, making them excellent choices for players who want poise without committing to full heavy load.**

# **📊 7: Elemental negation analysis (magic, fire, lightning, holy)**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    negation_magic,
    negation_fire,
    negation_lightning,
    negation_holy,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY negation_magic DESC
LIMIT 30;

armor_name armor_type negation_magic negation_fire negation_lightning negation_holy weight Azur's Glintstone Robe chest armor 15.4 12.8 13.0 13.8 7.1 Lusat's Robe chest armor 15.4 12.8 13.3 13.5 7.1 Preceptor's Long Gown chest armor 14.5 13.3 12.8 12.8 6.3 Dryleaf Robe chest armor 14.1 11.9 12.6 14.1 5.1 Gaius's Armor chest armor 14.1 13.8 13.8 13.8 20.1 Royal Knight Armor chest armor 14.1 13.3 12.4 13.0 15.5 Royal Knight Armor (Altered) chest armor 13.9 13.1 12.2 12.8 15.0 Finger Maiden Robe chest armor 13.8 13.1 13.3 13.8 4.6 Lazuli Robe chest armor 13.8 11.9 12.4 12.8 4.1 Dryleaf Robe (Altered) chest armor 13.8 11.4 12.4 13.8 4.1 Queen's Robe chest armor 13.8 12.6 12.8 13.3 5.1 Braided Cord Robe chest armor 13.6 11.9 12.9 14.0 3.4 Banished Knight Armor chest armor 13.5 13.5 13.0 13.3 17.5 Veteran's Armor chest armor 13.5 14.1 12.8 13.3 18.9 Astrologer Robe chest armor 13.5 13.0 13.3 13.0 6.3 Sage Robe chest armor 13.5 12.6 13.0 13.5 5.1 Radahn's Lion Armor chest armor 13.5 14.1 12.6 13.5 17.5 Fire Prelate Armor chest armor 13.5 19.8 13.0 13.3 24.7 Lionel's Armor chest armor 13.5 14.9 13.0 13.5 21.2 Prophet Robe chest armor 13.5 12.6 13.0 13.0 6.3 Raya Lucarian Robe chest armor 13.5 12.6 12.4 12.8 4.1 Verdigris Armor chest armor 13.5 13.5 14.5 13.3 25.9 Scaled Armor chest armor 13.5 14.1 13.0 13.5 16.0 High Page Clothes chest armor 13.5 12.8 13.0 13.2 5.3 Snow Witch Robe chest armor 13.4 13.4 13.2 13.4 5.5 Traveling Maiden Robe chest armor 13.4 12.9 13.1 13.4 4.9 Corhyn's Robe chest armor 13.3 12.8 12.8 13.5 6.3 Bull-Goat Armor chest armor 13.3 13.5 14.9 12.8 26.5 Sanguine Noble Robe chest armor 13.3 11.9 13.0 13.5 5.1 Noble's Traveling Garb chest armor 13.3 13.5 13.0 13.3 6.3

**The strongest magic-negating armors are mostly light to mid-weight robes, led by Azur’s Glintstone Robe and Lusat’s Robe, both offering the highest magic negation at 15.4 while staying relatively light at 7.1 weight. Other top performers like the Preceptor’s Long Gown, Dryleaf Robe, and Queen’s Robe continue this trend, providing high magic, fire, lightning, and holy negation without heavy load penalties. A few heavy armors—such as Gaius’s Armor, Banished Knight Armor, and Lionel’s Armor—appear on the list, showing strong all-around defenses but at much higher weights. Overall, the data shows that robes and lighter chest pieces offer the best magic defense per weight, while heavy armors only match these values by significantly increasing load.**

# **📊 8: Best magic negation armors**

In [0]:
%sql
SELECT 
    a.armor_type,
    a.armor_name,
    a.negation_magic,
    a.weight
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN (
    SELECT armor_type, MAX(negation_magic) as max_magic
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
) max_vals ON a.armor_type = max_vals.armor_type AND a.negation_magic = max_vals.max_magic
ORDER BY a.negation_magic DESC;

armor_type armor_name negation_magic weight chest armor Azur's Glintstone Robe 15.4 7.1 chest armor Lusat's Robe 15.4 7.1 leg armor Preceptor's Trousers 8.3 3.9 helm Nox Mirrorhelm 6.7 7.5 gauntlets Preceptor's Gloves 3.6 2.1

**Across all armor categories, the highest magic negation pieces are dominated by lightweight caster gear. Azur's Glintstone Robe and Lusat's Robe tie for the strongest overall protection with 15.4 magic negation at just 7.1 weight, making chest robes the best option for maximizing magical defense. For leg armor, Preceptor's Trousers lead with 8.3 magic negation at 3.9 weight, while the Nox Mirrorhelm provides the best magic negation among helms with 6.7, though at a slightly heavier 7.5 weight. Finally, Preceptor’s Gloves stand out among gauntlets with 3.6 magic negation at only 2.1 weight. Overall, the best magic defense per armor type consistently comes from light, mage-oriented gear, offering strong protection without the burden of heavy load.**

# **📊9: Resistance analysis (immunity, robustness, focus, vitality)**

In [0]:
%sql
SELECT armor_name, armor_type, resistance_immunity, resistance_robustness, resistance_focus, resistance_vitality, weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_immunity DESC LIMIT 30;

armor_name armor_type resistance_immunity resistance_robustness resistance_focus resistance_vitality weight Mushroom Body chest armor 102.0 24.0 90.0 76.0 5.1 Verdigris Armor chest armor 82.0 97.0 32.0 45.0 25.9 Rotten Gravekeeper Cloak chest armor 76.0 32.0 42.0 42.0 6.3 Thiollier's Garb chest armor 71.0 28.0 63.0 38.0 7.1 Bull-Goat Armor chest armor 71.0 83.0 46.0 55.0 26.5 Guardian Garb (Full Bloom) chest armor 71.0 46.0 57.0 55.0 8.8 Shadow Militiaman Armor chest armor 69.0 29.0 38.0 38.0 7.1 Fire Prelate Armor chest armor 67.0 63.0 90.0 50.0 24.7 White Reed Armor chest armor 67.0 42.0 55.0 50.0 8.3 Thiollier's Garb (Altered) chest armor 67.0 25.0 60.0 35.0 6.3 Omenkiller Robe chest armor 67.0 28.0 46.0 46.0 7.1 Tree Sentinel Armor chest armor 67.0 90.0 42.0 46.0 18.9 Omen Armor chest armor 67.0 63.0 57.0 76.0 23.1 Gaius's Armor chest armor 67.0 83.0 55.0 55.0 20.1 Vulgar Militia Armor chest armor 67.0 35.0 50.0 50.0 7.7 Ronin's Armor chest armor 66.0 48.0 52.0 57.0 8.5 Depraved Perfumer Robe chest armor 65.0 22.0 55.0 80.0 5.0 Rotten Gravekeeper Cloak (Altered) chest armor 64.0 25.0 34.0 34.0 5.1 Night Maiden Armor chest armor 63.0 38.0 55.0 55.0 8.3 Ivory-Draped Tabard chest armor 63.0 42.0 55.0 57.0 8.3 Bloodsoaked Tabard chest armor 63.0 38.0 55.0 50.0 8.3 Depraved Perfumer Robe (Altered) chest armor 63.0 18.0 48.0 63.0 4.0 Nox Monk Armor chest armor 63.0 42.0 50.0 50.0 8.1 Confessor Armor chest armor 63.0 50.0 50.0 55.0 8.3 Fur Raiment chest armor 63.0 42.0 55.0 38.0 5.1 Leather-Draped Tabard chest armor 63.0 42.0 55.0 55.0 8.3 Lord of Blood's Robe chest armor 63.0 23.0 55.0 55.0 8.1 Veteran's Armor chest armor 63.0 90.0 46.0 46.0 18.9 Chain-Draped Tabard chest armor 63.0 42.0 55.0 55.0 8.3 Cleanrot Armor chest armor 63.0 67.0 28.0 32.0 15.0

**The top 30 armors for immunity resistance are almost entirely chest pieces, showing that major body armor dominates this category. The Mushroom Body stands out with an exceptionally high 102 immunity at just 5.1 weight, making it by far the most efficient for resisting status effects. Other notable chest armors include Verdigris Armor (82 immunity, 25.9 weight) and Rotten Gravekeeper Cloak (76 immunity, 6.3 weight), which provide strong protection but vary significantly in weight. Several heavy armors like Bull-Goat Armor, Fire Prelate Armor, and Gaius’s Armor offer high immunity in combination with robust and vitality defenses, though at much higher weights. Overall, the dataset highlights that light special-purpose chest pieces maximize immunity, while heavier armors trade efficiency for all-around durability.**

In [0]:
%sql
SELECT armor_name, armor_type, resistance_immunity, negation_physical, weight FROM eldenringcatalog.gold.fact_armor_stats ORDER BY resistance_immunity DESC LIMIT 20;

armor_name armor_type resistance_immunity negation_physical weight Mushroom Body chest armor 102.0 6.1 5.1 Verdigris Armor chest armor 82.0 20.4 25.9 Rotten Gravekeeper Cloak chest armor 76.0 6.7 6.3 Guardian Garb (Full Bloom) chest armor 71.0 10.2 8.8 Thiollier's Garb chest armor 71.0 8.8 7.1 Bull-Goat Armor chest armor 71.0 20.4 26.5 Shadow Militiaman Armor chest armor 69.0 7.1 7.1 Thiollier's Garb (Altered) chest armor 67.0 8.0 6.3 Tree Sentinel Armor chest armor 67.0 18.7 18.9 Gaius's Armor chest armor 67.0 16.8 20.1 Vulgar Militia Armor chest armor 67.0 8.8 7.7 Omenkiller Robe chest armor 67.0 8.8 7.1 White Reed Armor chest armor 67.0 9.5 8.3 Omen Armor chest armor 67.0 18.3 23.1 Fire Prelate Armor chest armor 67.0 19.2 24.7 Ronin's Armor chest armor 66.0 10.5 8.5 Depraved Perfumer Robe chest armor 65.0 6.0 5.0 Rotten Gravekeeper Cloak (Altered) chest armor 64.0 6.1 5.1 Chain-Draped Tabard chest armor 63.0 9.5 8.3 Cleanrot Armor chest armor 63.0 14.6 15.0

In [0]:
%sql
SELECT armor_name, armor_type, resistance_robustness, negation_physical, weight FROM eldenringcatalog.gold.fact_armor_stats ORDER BY resistance_robustness DESC LIMIT 20;

armor_name armor_type resistance_robustness negation_physical weight Radahn's Lion Armor chest armor 99.0 18.7 17.5 Lionel's Armor chest armor 99.0 17.5 21.2 Verdigris Armor chest armor 97.0 20.4 25.9 Tree Sentinel Armor chest armor 90.0 18.7 18.9 Lionel's Armor (Altered) chest armor 90.0 17.1 20.2 Veteran's Armor chest armor 90.0 18.7 18.9 Beast Champion Armor chest armor 90.0 17.5 17.5 Bull-Goat Armor chest armor 83.0 20.4 26.5 Gaius's Armor chest armor 83.0 16.8 20.1 Scaled Armor chest armor 83.0 16.0 16.0 Banished Knight Armor chest armor 83.0 18.7 17.5 Radahn's Lion Armor (Altered) chest armor 82.0 18.5 16.4 Blaidd's Armor chest armor 76.0 14.6 13.7 Malformed Dragon Armor chest armor 76.0 16.8 16.0 Beast Champion Armor (Altered) chest armor 75.0 17.2 16.4 Tree Sentinel Armor (Altered) chest armor 75.0 18.5 18.1 Veteran's Armor (Altered) chest armor 75.0 18.1 17.2 Briar Armor chest armor 71.0 12.9 11.8 Crucible Tree Armor chest armor 71.0 17.7 15.5 Crucible Axe Armor chest armor 71.0 17.5 15.5

In [0]:
%sql
SELECT armor_name, armor_type, resistance_focus, negation_magic, weight FROM eldenringcatalog.gold.fact_armor_stats ORDER BY resistance_focus DESC LIMIT 20;

armor_name armor_type resistance_focus negation_magic weight Corhyn's Robe chest armor 107.0 13.3 6.3 Braided Cord Robe chest armor 104.0 13.6 3.4 Azur's Glintstone Robe chest armor 99.0 15.4 7.1 Armor of Solitude chest armor 99.0 13.3 24.0 Mushroom Body chest armor 90.0 13.3 5.1 Armor of Solitude (Altered) chest armor 90.0 13.0 23.2 Astrologer Robe chest armor 90.0 13.5 6.3 Fire Prelate Armor chest armor 90.0 13.5 24.7 Preceptor's Long Gown chest armor 83.0 14.5 6.3 Prophet Robe chest armor 83.0 13.5 6.3 Page Garb chest armor 83.0 12.8 5.1 Sanguine Noble Robe chest armor 83.0 13.3 5.1 Lusat's Robe chest armor 83.0 15.4 7.1 Ansbach's Attire chest armor 76.0 13.0 5.1 Dryleaf Robe chest armor 76.0 14.1 5.1 Commoner's Garb chest armor 76.0 12.8 5.1 Noble's Traveling Garb chest armor 76.0 13.3 6.3 Traveling Maiden Robe chest armor 76.0 13.4 4.9 Commoner's Simple Garb chest armor 76.0 13.0 5.1 Millicent's Tunic chest armor 76.0 13.0 8.3

In [0]:
%sql
SELECT armor_name, armor_type, resistance_vitality, negation_holy, weight FROM eldenringcatalog.gold.fact_armor_stats ORDER BY resistance_vitality DESC LIMIT 20;

armor_name armor_type resistance_vitality negation_holy weight Fia's Robe chest armor 108.0 13.0 5.1 Deathbed Dress chest armor 107.0 12.4 3.2 Lusat's Robe chest armor 99.0 13.5 7.1 Fia's Robe (Altered) chest armor 93.0 12.6 3.2 Ansbach's Attire chest armor 90.0 13.0 5.1 Prophet Robe chest armor 90.0 13.0 6.3 Commoner's Simple Garb chest armor 83.0 12.8 5.1 Commoner's Garb chest armor 83.0 12.8 5.1 Noble's Traveling Garb chest armor 83.0 13.3 6.3 Azur's Glintstone Robe chest armor 83.0 13.8 7.1 Traveling Maiden Robe chest armor 83.0 13.4 4.9 Page Garb chest armor 83.0 13.0 5.1 Queen's Robe chest armor 83.0 13.3 5.1 Godskin Noble Robe chest armor 83.0 14.1 6.3 Fell Omen Cloak chest armor 83.0 12.8 5.1 Finger Robe chest armor 82.0 12.8 5.1 Depraved Perfumer Robe chest armor 80.0 12.7 5.0 Mushroom Body chest armor 76.0 13.0 5.1 Astrologer Robe chest armor 76.0 13.0 6.3 Millicent's Tunic chest armor 76.0 13.0 8.3

**The data shows that chest armor dominates all major resistance categories in Elden Ring, consistently outperforming other armor types in immunity, robustness, focus, vitality, and both physical and magical negation. Lightweight specialty pieces like the Mushroom Body, Braided Cord Robe, and Azur’s Glintstone Robe excel in immunity, focus, and vitality, offering strong defenses at very low weights. In contrast, heavy armors such as Verdigris Armor, Bull-Goat Armor, Tree Sentinel Armor, and Fire Prelate Armor provide top-tier robustness and physical negation but at the cost of much higher weight. Certain versatile pieces, like Lusat’s Robe and Fia’s Robe, balance high resistances across multiple categories with moderate weight, making them ideal for hybrid defensive builds. Overall, the dataset illustrates a clear trade-off: light chest pieces maximize efficiency and magical/poise defenses, while heavy chest armors prioritize raw physical and robustness protection.**

In [0]:
%sql
SELECT armor_name, armor_type,
    (negation_physical + negation_strike + negation_slash + negation_pierce + negation_magic + negation_fire + negation_lightning + negation_holy) AS total_negation,
    (resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) AS total_resistance,
    (negation_physical + negation_strike + negation_slash + negation_pierce + negation_magic + negation_fire + negation_lightning + negation_holy) + 
    ((resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) * 0.5) AS defense_score,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY defense_score DESC LIMIT 30;

armor_name armor_type total_negation total_resistance defense_score weight Fire Prelate Armor chest armor 132.4 315.0 289.9 24.7 Armor of Solitude chest armor 127.3 317.0 285.8 24.0 Verdigris Armor chest armor 131.3 303.0 282.8 25.9 Bull-Goat Armor chest armor 131.7 302.0 282.7 26.5 Omen Armor chest armor 126.9 306.0 279.9 23.1 Gaius's Armor chest armor 126.89999999999999 298.0 275.9 20.1 Lionel's Armor chest armor 127.60000000000001 294.0 274.6 21.2 Armor of Solitude (Altered) chest armor 123.49999999999999 296.0 271.5 23.2 Tree Sentinel Armor chest armor 126.69999999999999 284.0 268.7 18.9 Veteran's Armor chest armor 125.6 283.0 267.1 18.9 Fire Prelate Armor (Altered) chest armor 127.39999999999999 271.0 262.9 23.6 Radahn's Lion Armor chest armor 123.19999999999999 273.0 259.7 17.5 Lionel's Armor (Altered) chest armor 124.39999999999998 268.0 258.4 20.2 Beast Champion Armor chest armor 123.89999999999999 261.0 254.39999999999998 17.5 Banished Knight Armor chest armor 123.7 257.0 252.2 17.5 Tree Sentinel Armor (Altered) chest armor 124.5 239.0 244.0 18.1 Veteran's Armor (Altered) chest armor 122.6 240.0 242.6 17.2 Scaled Armor chest armor 116.8 249.0 241.3 16.0 Malformed Dragon Armor chest armor 118.89999999999999 240.0 238.89999999999998 16.0 Radahn's Lion Armor (Altered) chest armor 120.8 233.0 237.3 16.4 Beast Champion Armor (Altered) chest armor 122.50000000000001 225.0 235.0 16.4 Banished Knight Armor (Altered) chest armor 120.6 220.0 230.6 16.5 Crucible Tree Armor chest armor 115.70000000000002 224.0 227.70000000000002 15.5 Crucible Axe Armor chest armor 115.7 224.0 227.7 15.5 Royal Knight Armor chest armor 117.5 211.0 223.0 15.5 Cleanrot Armor chest armor 111.8 217.0 220.3 15.0 Lusat's Robe chest armor 83.2 269.0 217.7 7.1 Azur's Glintstone Robe chest armor 81.6 272.0 217.6 7.1 Royal Knight Armor (Altered) chest armor 115.89999999999999 198.0 214.89999999999998 15.0 Mushroom Body chest armor 65.6 297.0 214.1 5.1

**The highest overall defense scores are dominated by heavy chest armors, with Fire Prelate Armor, Armor of Solitude, Verdigris Armor, and Bull-Goat Armor leading the list. These armors combine very high physical and elemental negation with strong overall resistances, giving them the top defense scores of around 280–290, though they come with substantial weight (20–26.5). Mid-tier chest armors like Omen Armor, Gaius’s Armor, and Lionel’s Armor also provide excellent all-around protection, balancing solid negation and resistance with slightly lower weight. Lightweight magical or specialty robes, such as Lusat’s Robe, Azur’s Glintstone Robe, and Mushroom Body, achieve respectable defense scores (~217) despite their low weight, highlighting their efficiency for mobility-focused builds. Overall, heavy chest armor maximizes total defense, while light mage-focused gear offers a highly weight-efficient alternative with decent protection.**

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise FROM eldenringcatalog.gold.fact_armor_stats WHERE armor_type = 'helm' ORDER BY weight LIMIT 3;

armor_name weight negation_physical resistance_poise Commoner's Headband (Altered) 0.7 0.1 1.0 Dancer's Hood 1.0 0.9 1.0 Leather Headband 1.0 1.4 2.0

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise FROM eldenringcatalog.gold.fact_armor_stats WHERE armor_type = 'helm' ORDER BY weight DESC LIMIT 3;

armor_name weight negation_physical resistance_poise Greatjar 12.3 8.0 14.0 Pumpkin Helm 12.3 7.0 11.0 Bull-Goat Helm 11.3 7.5 15.0

**The lightest helms are extremely mobility-friendly but offer minimal protection. For example, the Commoner's Headband (Altered) weighs only 0.7, providing almost no physical negation (0.1) and minimal poise (1). Similarly, Dancer’s Hood and Leather Headband are light (1 weight) but give only modest physical defense (0.9–1.4) and poise (1–2).**

**At the other end, heavy helms deliver substantial protection at a cost of mobility. The Greatjar and Pumpkin Helm weigh over 12, providing strong physical negation (7–8) and high poise (11–14), while the Bull-Goat Helm balances slightly less weight (11.3) with even higher poise (15) and solid physical defense (7.5).**

**In short, helm choice is a trade-off: light helms maximize speed and mobility with minimal defense, while heavy helms significantly boost physical protection and poise at the expense of weight.**

In [0]:
%sql
SELECT 
    CASE WHEN a.is_dlc = 1 THEN 'DLC' ELSE 'Base Game' END AS content_type,
    COUNT(*) AS armor_count,
    ROUND(AVG(s.weight), 2) AS avg_weight,
    ROUND(AVG(s.negation_physical), 2) AS avg_physical,
    ROUND(AVG(s.resistance_poise), 2) AS avg_poise
FROM eldenringcatalog.gold.dim_armors a
INNER JOIN eldenringcatalog.gold.fact_armor_stats s ON a.armor_name = s.armor_name
GROUP BY a.is_dlc;

content_type armor_count avg_weight avg_physical avg_poise DLC 145 6.02 6.09 10.82 Base Game 578 5.99 6.12 10.56

**Across Elden Ring, DLC and base game armors are very similar on average. DLC armors (145 pieces) have an average weight of 6.02, physical negation of 6.09, and poise of 10.82, while base game armors (578 pieces) average 5.99 weight, 6.12 physical negation, and 10.56 poise. This indicates that DLC armors maintain comparable defensive performance to base game gear, with only a slight increase in poise on average.**

In [0]:
%sql
SELECT armor_name, armor_type, weight,
       (negation_physical + negation_strike + negation_slash + negation_pierce +
        negation_magic + negation_fire + negation_lightning + negation_holy) AS total_negation,
       (resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) AS total_resistance,
       ((negation_physical + negation_strike + negation_slash + negation_pierce +
         negation_magic + negation_fire + negation_lightning + negation_holy) + 
         (resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise)*0.5) AS defense_score
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight BETWEEN 10 AND 25
ORDER BY defense_score DESC
LIMIT 20;


armor_name armor_type weight total_negation total_resistance defense_score Fire Prelate Armor chest armor 24.7 132.4 315.0 289.9 Armor of Solitude chest armor 24.0 127.3 317.0 285.8 Omen Armor chest armor 23.1 126.9 306.0 279.9 Gaius's Armor chest armor 20.1 126.89999999999999 298.0 275.9 Lionel's Armor chest armor 21.2 127.60000000000001 294.0 274.6 Armor of Solitude (Altered) chest armor 23.2 123.49999999999999 296.0 271.5 Tree Sentinel Armor chest armor 18.9 126.69999999999999 284.0 268.7 Veteran's Armor chest armor 18.9 125.6 283.0 267.1 Fire Prelate Armor (Altered) chest armor 23.6 127.39999999999999 271.0 262.9 Radahn's Lion Armor chest armor 17.5 123.19999999999999 273.0 259.7 Lionel's Armor (Altered) chest armor 20.2 124.39999999999998 268.0 258.4 Beast Champion Armor chest armor 17.5 123.89999999999999 261.0 254.39999999999998 Banished Knight Armor chest armor 17.5 123.7 257.0 252.2 Tree Sentinel Armor (Altered) chest armor 18.1 124.5 239.0 244.0 Veteran's Armor (Altered) chest armor 17.2 122.6 240.0 242.6 Scaled Armor chest armor 16.0 116.8 249.0 241.3 Malformed Dragon Armor chest armor 16.0 118.89999999999999 240.0 238.89999999999998 Radahn's Lion Armor (Altered) chest armor 16.4 120.8 233.0 237.3 Beast Champion Armor (Altered) chest armor 16.4 122.50000000000001 225.0 235.0 Banished Knight Armor (Altered) chest armor 16.5 120.6 220.0 230.6

**the balanced armor sets with moderate weight between 10 and 25 units, prioritizing overall defense efficiency. The top entries are predominantly chest armor pieces, with Fire Prelate Armor leading with a defense score of 289.9, followed closely by Armor of Solitude (285.8) and Omen Armor (279.9). These armors combine high total negation against physical, elemental, and magical damage with substantial resistances in immunity, robustness, focus, vitality, and poise. Other notable armors include Gaius’s Armor, Lionel’s Armor, and Tree Sentinel Armor, all offering strong protection without excessively burdening the wearer. Overall, this selection demonstrates a careful balance between weight and defensive capabilities, making them ideal for players seeking robust protection while maintaining moderate mobility.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight <= 5
ORDER BY weight ASC, (negation_physical + resistance_poise) DESC
LIMIT 20;

armor_name armor_type weight negation_physical resistance_poise Commoner's Headband (Altered) helm 0.7 0.1 1.0 Gold Bracelets gauntlets 0.8 1.0 1.0 Braided Arm Wraps gauntlets 0.8 1.0 1.0 Brave's Bracer gauntlets 0.8 0.6 1.0 Circlet of Light helm 1.0 2.8 5.0 Leather Headband helm 1.0 1.4 2.0 St. Trina's Blossom helm 1.0 0.9 2.0 Brave's Cord Circlet helm 1.0 0.9 2.0 Dancer's Hood helm 1.0 0.9 1.0 Alberich's Pointed Hat (Altered) helm 1.0 0.9 1.0 Prophet Blindfold helm 1.0 0.2 1.0 Festive Hood (Altered) helm 1.0 0.2 1.0 Ascetic's Wrist Guards gauntlets 1.1 1.5 2.0 Battlemage Manchettes gauntlets 1.1 1.0 2.0 Lusat's Manchettes gauntlets 1.1 1.3 1.0 Azur's Manchettes gauntlets 1.1 1.0 1.0 Sorcerer Manchettes gauntlets 1.1 1.0 1.0 Millicent's Gloves gauntlets 1.1 1.0 1.0 Errant Sorcerer Manchettes gauntlets 1.1 0.6 1.0 Aristocrat Headband helm 1.2 1.9 2.0

**the lightweight armor pieces, ideal for players prioritizing mobility and speed while still retaining modest protection. The lightest items include Commoner’s Headband (Altered) at just 0.7 weight and gold or braided gauntlets at 0.8 weight, offering minimal physical negation and poise. Notably, Circlet of Light stands out with 2.8 physical negation and 5 poise at only 1 weight, providing a remarkable balance of protection and lightness. Other helms like Leather Headband, St. Trina’s Blossom, and Brave’s Cord Circlet provide modest stats while keeping the weight extremely low. Gauntlets like Ascetic’s Wrist Guards and Battlemage Manchettes also contribute with light protection without hindering movement. Overall, these armor pieces are best suited for agile or caster-focused builds that benefit from minimal encumbrance while still gaining some defensive value.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight >= 20
ORDER BY (negation_physical + resistance_poise) DESC
LIMIT 20;

armor_name armor_type weight negation_physical resistance_poise Bull-Goat Armor chest armor 26.5 20.4 47.0 Verdigris Armor chest armor 25.9 20.4 47.0 Fire Prelate Armor chest armor 24.7 19.2 45.0 Armor of Solitude chest armor 24.0 18.7 45.0 Fire Prelate Armor (Altered) chest armor 23.6 18.7 43.0 Omen Armor chest armor 23.1 18.3 43.0 Armor of Solitude (Altered) chest armor 23.2 18.3 43.0 Lionel's Armor chest armor 21.2 17.5 40.0 Lionel's Armor (Altered) chest armor 20.2 17.1 38.0 Gaius's Armor chest armor 20.1 16.8 38.0

**The heavy armor sets, which are designed for maximum protection and high poise, making them ideal for tanky, frontline playstyles. Leading the list are Bull-Goat Armor and Verdigris Armor, each weighing over 25 units while providing a whopping 20.4 physical negation and 47 poise, making them excellent for withstanding powerful enemy attacks. Other notable heavy sets include Fire Prelate Armor and Armor of Solitude, both offering nearly 19 physical negation and 45 poise at slightly lower weights around 24–24.7. Additional options like Omen Armor and Lionel's Armor provide strong defensive stats for slightly more maneuverable heavy builds. Overall, these armors are best suited for players who prioritize survivability over mobility, allowing them to endure massive damage while maintaining stability in combat.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_physical, negation_magic
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY negation_physical DESC, negation_magic DESC
LIMIT 20;

armor_name armor_type weight negation_physical negation_magic Verdigris Armor chest armor 25.9 20.4 13.5 Bull-Goat Armor chest armor 26.5 20.4 13.3 Fire Prelate Armor chest armor 24.7 19.2 13.5 Veteran's Armor chest armor 18.9 18.7 13.5 Radahn's Lion Armor chest armor 17.5 18.7 13.5 Banished Knight Armor chest armor 17.5 18.7 13.5 Armor of Solitude chest armor 24.0 18.7 13.3 Fire Prelate Armor (Altered) chest armor 23.6 18.7 13.3 Tree Sentinel Armor chest armor 18.9 18.7 13.0 Radahn's Lion Armor (Altered) chest armor 16.4 18.5 13.3 Tree Sentinel Armor (Altered) chest armor 18.1 18.5 12.6 Banished Knight Armor (Altered) chest armor 16.5 18.3 13.3 Armor of Solitude (Altered) chest armor 23.2 18.3 13.0 Omen Armor chest armor 23.1 18.3 13.0 Veteran's Armor (Altered) chest armor 17.2 18.1 13.1 Crucible Tree Armor chest armor 15.5 17.7 13.0 Lionel's Armor chest armor 21.2 17.5 13.5 Beast Champion Armor chest armor 17.5 17.5 13.0 Crucible Axe Armor chest armor 15.5 17.5 13.0 Beast Champion Armor (Altered) chest armor 16.4 17.2 12.9

**the most physically resilient armors in Elden Ring while also considering their magical resistance. At the top are heavy-hitting chest armors like Verdigris Armor and Bull-Goat Armor, both offering 20.4 physical negation and solid magic negation values of 13.3–13.5, making them highly reliable for tank-focused builds. Following closely are Fire Prelate Armor, Veteran’s Armor, and Radahn’s Lion Armor, providing slightly lower physical negation but maintaining strong magical defense, ideal for hybrid durability against both physical and elemental attacks. Other noteworthy armors, including Armor of Solitude, Tree Sentinel Armor, and Banished Knight Armor, balance high physical protection with moderate magic negation, giving players flexibility to withstand diverse combat threats. Overall, these armors cater to players prioritizing maximum physical resilience without sacrificing moderate magical protection, suitable for aggressive frontline roles or boss encounters.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_magic, negation_fire, negation_lightning
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY negation_magic DESC
LIMIT 20;

armor_name armor_type weight negation_magic negation_fire negation_lightning Lusat's Robe chest armor 7.1 15.4 12.8 13.3 Azur's Glintstone Robe chest armor 7.1 15.4 12.8 13.0 Preceptor's Long Gown chest armor 6.3 14.5 13.3 12.8 Royal Knight Armor chest armor 15.5 14.1 13.3 12.4 Dryleaf Robe chest armor 5.1 14.1 11.9 12.6 Gaius's Armor chest armor 20.1 14.1 13.8 13.8 Royal Knight Armor (Altered) chest armor 15.0 13.9 13.1 12.2 Queen's Robe chest armor 5.1 13.8 12.6 12.8 Dryleaf Robe (Altered) chest armor 4.1 13.8 11.4 12.4 Finger Maiden Robe chest armor 4.6 13.8 13.1 13.3 Lazuli Robe chest armor 4.1 13.8 11.9 12.4 Braided Cord Robe chest armor 3.4 13.6 11.9 12.9 Verdigris Armor chest armor 25.9 13.5 13.5 14.5 Prophet Robe chest armor 6.3 13.5 12.6 13.0 Radahn's Lion Armor chest armor 17.5 13.5 14.1 12.6 Astrologer Robe chest armor 6.3 13.5 13.0 13.3 Raya Lucarian Robe chest armor 4.1 13.5 12.6 12.4 Sage Robe chest armor 5.1 13.5 12.6 13.0 Fire Prelate Armor chest armor 24.7 13.5 19.8 13.0 Banished Knight Armor chest armor 17.5 13.5 13.5 13.0

**the armors with the highest magic defense in Elden Ring, ideal for caster-focused or hybrid builds. At the top are lightweight robes such as Lusat's Robe and Azur's Glintstone Robe, both offering 15.4 magic negation with moderate fire and lightning resistances, perfect for maximizing spell defense while keeping weight low. Close contenders include Preceptor's Long Gown and Dryleaf Robe, which combine high magic negation with decent elemental resistances. Heavy armors like Royal Knight Armor and Gaius's Armor provide slightly lower magic negation (13.9–14.1) but offer higher overall physical defense due to their weight and durability. Notably, Fire Prelate Armor stands out with 19.8 fire negation, making it a specialized choice against fire-heavy encounters. Overall, these armors cater to players who prioritize strong magical defense, with options ranging from light, agile sets to heavier, more protective variants, allowing flexibility for different combat scenarios.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, resistance_poise, negation_physical
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_poise DESC, weight DESC
LIMIT 20;

armor_name armor_type weight resistance_poise negation_physical Bull-Goat Armor chest armor 26.5 47.0 20.4 Verdigris Armor chest armor 25.9 47.0 20.4 Fire Prelate Armor chest armor 24.7 45.0 19.2 Armor of Solitude chest armor 24.0 45.0 18.7 Fire Prelate Armor (Altered) chest armor 23.6 43.0 18.7 Armor of Solitude (Altered) chest armor 23.2 43.0 18.3 Omen Armor chest armor 23.1 43.0 18.3 Lionel's Armor chest armor 21.2 40.0 17.5 Tree Sentinel Armor chest armor 18.9 39.0 18.7 Lionel's Armor (Altered) chest armor 20.2 38.0 17.1 Gaius's Armor chest armor 20.1 38.0 16.8 Veteran's Armor chest armor 18.9 38.0 18.7 Tree Sentinel Armor (Altered) chest armor 18.1 35.0 18.5 Veteran's Armor (Altered) chest armor 17.2 35.0 18.1 Beast Champion Armor chest armor 17.5 34.0 17.5 Banished Knight Armor chest armor 17.5 34.0 18.7 Radahn's Lion Armor chest armor 17.5 34.0 18.7 Beast Champion Armor (Altered) chest armor 16.4 34.0 17.2 Banished Knight Armor (Altered) chest armor 16.5 33.0 18.3 Radahn's Lion Armor (Altered) chest armor 16.4 33.0 18.5

**the armors with the highest poise, making them ideal for players who want to tank hits and maintain stability during combat. Leading the list are heavysets like Bull-Goat Armor and Verdigris Armor, both with a poise of 47 and high physical negation (20.4), offering maximum resilience against stagger and physical attacks. Other strong contenders include Fire Prelate Armor and Armor of Solitude, which balance high poise (43–45) with substantial physical defense (18.3–19.2) while maintaining manageable weight for their class. Mid-weight armors such as Tree Sentinel Armor, Veteran’s Armor, and Lionel’s Armor provide a combination of good poise (33–39) and solid physical protection, suitable for players who want a compromise between mobility and defensive durability. Overall, these sets are optimal for tanking roles, heavy-hitters, or anyone prioritizing stability and stagger resistance in melee-heavy encounters.**

In [0]:
%sql
SELECT armor_name, armor_type, weight, negation_magic, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight <= 5
ORDER BY negation_magic DESC, resistance_poise DESC
LIMIT 20;

armor_name armor_type weight negation_magic resistance_poise Finger Maiden Robe chest armor 4.6 13.8 10.0 Dryleaf Robe (Altered) chest armor 4.1 13.8 8.0 Lazuli Robe chest armor 4.1 13.8 8.0 Braided Cord Robe chest armor 3.4 13.6 5.0 Raya Lucarian Robe chest armor 4.1 13.5 7.0 Traveling Maiden Robe chest armor 4.9 13.4 10.0 Traveler's Clothes chest armor 4.1 13.3 8.0 Depraved Perfumer Robe chest armor 5.0 13.2 8.0 High Page Clothes (Altered) chest armor 4.3 13.2 8.0 Snow Witch Robe (Altered) chest armor 4.6 13.2 7.0 Traveling Maiden Robe (Altered) chest armor 4.3 13.1 8.0 Battlemage Robe chest armor 4.1 13.0 10.0 Finger Maiden Robe (Altered) chest armor 3.7 13.0 8.0 Errant Sorcerer Robe chest armor 4.1 13.0 8.0 Depraved Perfumer Robe (Altered) chest armor 4.0 13.0 7.0 Consort's Robe chest armor 4.1 13.0 7.0 High Priest Robe chest armor 4.2 12.8 10.0 Ansbach's Attire (Altered) chest armor 4.1 12.8 10.0 Brave's Battlewear chest armor 4.1 12.8 8.0 Upper-Class Robe chest armor 4.1 12.8 8.0

**The best light armors for players who want high magical protection without sacrificing mobility. The top choices are robes like Finger Maiden Robe, Dryleaf Robe (Altered), and Lazuli Robe, which weigh under 5 units and provide exceptional magic negation (13–13.8) while maintaining some poise (5–10). These sets are ideal for mages, hybrid spellcasters, or agile builds, allowing them to move quickly in combat while mitigating magic damage. Other notable options include Traveling Maiden Robe, Battlemage Robe, and High Priest Robe, offering a good balance of light weight, respectable poise, and strong magical defense. Overall, these armors are perfect for players prioritizing spell resistance, mobility, and light loadouts, making them highly effective in magical-heavy encounters or boss fights where dodging is key.**

In [0]:
%sql
SELECT armor_name, armor_type, weight,
       (negation_physical + negation_strike + negation_slash + negation_pierce +
        negation_magic + negation_fire + negation_lightning + negation_holy +
        resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) AS total_defense
FROM eldenringcatalog.gold.fact_armor_stats
WHERE weight >= 20
ORDER BY total_defense DESC
LIMIT 20;

armor_name armor_type weight total_defense Fire Prelate Armor chest armor 24.7 447.4 Armor of Solitude chest armor 24.0 444.3 Verdigris Armor chest armor 25.9 434.3 Bull-Goat Armor chest armor 26.5 433.7 Omen Armor chest armor 23.1 432.9 Gaius's Armor chest armor 20.1 424.9 Lionel's Armor chest armor 21.2 421.6 Armor of Solitude (Altered) chest armor 23.2 419.5 Fire Prelate Armor (Altered) chest armor 23.6 398.4 Lionel's Armor (Altered) chest armor 20.2 392.4

**The heaviest and most defensive armor sets in Elden Ring, focusing on pieces weighing 20 units or more. Leading the list is Fire Prelate Armor (24.7 weight) with a total defense score of 447.4, followed closely by Armor of Solitude (444.3) and Verdigris Armor (434.3). These armors combine high physical and elemental negation with strong resistances across immunity, robustness, focus, vitality, and poise, making them ideal for players seeking maximum survivability in frontline combat. Other notable heavy sets include Bull-Goat Armor, Omen Armor, and Gaius's Armor, offering both durability and significant poise, which are critical for tanking damage-heavy bosses. These armors excel in heavy, high-defense builds, though their weight may reduce mobility, favoring a slow, resilient playstyle that trades speed for survivability.**

# **📊 10: Best poison/rot resistance (immunity)**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    resistance_immunity,
    negation_physical,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_immunity DESC
LIMIT 20;

armor_name armor_type resistance_immunity negation_physical weight Mushroom Body chest armor 102.0 6.1 5.1 Verdigris Armor chest armor 82.0 20.4 25.9 Rotten Gravekeeper Cloak chest armor 76.0 6.7 6.3 Guardian Garb (Full Bloom) chest armor 71.0 10.2 8.8 Thiollier's Garb chest armor 71.0 8.8 7.1 Bull-Goat Armor chest armor 71.0 20.4 26.5 Shadow Militiaman Armor chest armor 69.0 7.1 7.1 Thiollier's Garb (Altered) chest armor 67.0 8.0 6.3 Tree Sentinel Armor chest armor 67.0 18.7 18.9 Gaius's Armor chest armor 67.0 16.8 20.1 Vulgar Militia Armor chest armor 67.0 8.8 7.7 Omenkiller Robe chest armor 67.0 8.8 7.1 White Reed Armor chest armor 67.0 9.5 8.3 Omen Armor chest armor 67.0 18.3 23.1 Fire Prelate Armor chest armor 67.0 19.2 24.7 Ronin's Armor chest armor 66.0 10.5 8.5 Depraved Perfumer Robe chest armor 65.0 6.0 5.0 Rotten Gravekeeper Cloak (Altered) chest armor 64.0 6.1 5.1 Chain-Draped Tabard chest armor 63.0 9.5 8.3 Cleanrot Armor chest armor 63.0 14.6 15.0

**The top armors in Elden Ring with the highest immunity resistance, which primarily helps reduce the effects of status ailments. Leading the list is the Mushroom Body (chest armor) with an immunity of 102, though its physical negation is moderate at 6.1 and it’s relatively light at 5.1 weight. Heavy hitters like Verdigris Armor (82 immunity, 20.4 physical negation, 25.9 weight) and Bull-Goat Armor (71 immunity, 20.4 physical negation, 26.5 weight) combine strong immunity with high physical protection, making them ideal for tanky builds. Other notable entries, such as Omen Armor and Fire Prelate Armor, provide balanced immunity and substantial negation, though they are heavier (over 20 weight). Light but high-immunity sets like Rotten Gravekeeper Cloak offer more mobility at the cost of lower physical defense. Overall, this list is essential for players who want status resistance prioritized while balancing weight and physical protection.**

# **📊 11: Best bleed/frost resistance (robustness)**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    resistance_robustness,
    negation_physical,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_robustness DESC
LIMIT 20

armor_name armor_type resistance_robustness negation_physical weight Radahn's Lion Armor chest armor 99.0 18.7 17.5 Lionel's Armor chest armor 99.0 17.5 21.2 Verdigris Armor chest armor 97.0 20.4 25.9 Tree Sentinel Armor chest armor 90.0 18.7 18.9 Lionel's Armor (Altered) chest armor 90.0 17.1 20.2 Veteran's Armor chest armor 90.0 18.7 18.9 Beast Champion Armor chest armor 90.0 17.5 17.5 Bull-Goat Armor chest armor 83.0 20.4 26.5 Gaius's Armor chest armor 83.0 16.8 20.1 Scaled Armor chest armor 83.0 16.0 16.0 Banished Knight Armor chest armor 83.0 18.7 17.5 Radahn's Lion Armor (Altered) chest armor 82.0 18.5 16.4 Blaidd's Armor chest armor 76.0 14.6 13.7 Malformed Dragon Armor chest armor 76.0 16.8 16.0 Beast Champion Armor (Altered) chest armor 75.0 17.2 16.4 Tree Sentinel Armor (Altered) chest armor 75.0 18.5 18.1 Veteran's Armor (Altered) chest armor 75.0 18.1 17.2 Briar Armor chest armor 71.0 12.9 11.8 Crucible Tree Armor chest armor 71.0 17.7 15.5 Crucible Axe Armor chest armor 71.0 17.5 15.5

**This query lists the top 20 Elden Ring armors with the highest robustness, which mainly reduces staggering and knockback effects. Leading the pack are Radahn’s Lion Armor and Lionel’s Armor (both 99 robustness), combining high physical negation (18.7 & 17.5) with moderate-to-heavy weight (17.5 and 21.2). Verdigris Armor stands out for having very high physical negation (20.4) but comes with a heavy weight (25.9). Heavy tanky sets like Bull-Goat Armor and Banished Knight Armor offer a balance of robustness and physical protection, suitable for high-poise, frontline builds. Medium-weight options, like Scaled Armor and Malformed Dragon Armor, provide decent robustness with lower weight, making them better for mobility-focused tanks. Overall, the list helps players prioritize survivability against stagger while considering weight and physical defense.**

# **📊 Question 12: Best madness/sleep resistance (focus)**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    resistance_focus,
    negation_magic,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_focus DESC
LIMIT 20;

armor_name armor_type resistance_focus negation_magic weight Corhyn's Robe chest armor 107.0 13.3 6.3 Braided Cord Robe chest armor 104.0 13.6 3.4 Azur's Glintstone Robe chest armor 99.0 15.4 7.1 Armor of Solitude chest armor 99.0 13.3 24.0 Mushroom Body chest armor 90.0 13.3 5.1 Armor of Solitude (Altered) chest armor 90.0 13.0 23.2 Astrologer Robe chest armor 90.0 13.5 6.3 Fire Prelate Armor chest armor 90.0 13.5 24.7 Preceptor's Long Gown chest armor 83.0 14.5 6.3 Prophet Robe chest armor 83.0 13.5 6.3 Page Garb chest armor 83.0 12.8 5.1 Sanguine Noble Robe chest armor 83.0 13.3 5.1 Lusat's Robe chest armor 83.0 15.4 7.1 Ansbach's Attire chest armor 76.0 13.0 5.1 Dryleaf Robe chest armor 76.0 14.1 5.1 Commoner's Garb chest armor 76.0 12.8 5.1 Noble's Traveling Garb chest armor 76.0 13.3 6.3 Traveling Maiden Robe chest armor 76.0 13.4 4.9 Commoner's Simple Garb chest armor 76.0 13.0 5.1 Millicent's Tunic chest armor 76.0 13.0 8.3

**The top armors by focus resistance in Elden Ring are mostly light to mid-weight robes, offering excellent protection against focus damage while maintaining mobility. Heavy options like Fire Prelate and Armor of Solitude provide high defense but reduce agility, making them ideal for tanky magic builds. Light armors such as Braided Cord, Lusat’s Robe, and Preceptor’s Long Gown favor rolling and fast casting, while balanced options like Astrologer Robe and Mushroom Body provide decent protection without sacrificing movement. Overall, players can choose between mobility-focused, tanky, or hybrid magic builds depending on weight and focus resistance priorities.**

# **📊 13: Best death blight resistance (vitality)**


In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    resistance_vitality,
    negation_holy,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY resistance_vitality DESC
LIMIT 20;

armor_name armor_type resistance_vitality negation_holy weight Fia's Robe chest armor 108.0 13.0 5.1 Deathbed Dress chest armor 107.0 12.4 3.2 Lusat's Robe chest armor 99.0 13.5 7.1 Fia's Robe (Altered) chest armor 93.0 12.6 3.2 Ansbach's Attire chest armor 90.0 13.0 5.1 Prophet Robe chest armor 90.0 13.0 6.3 Commoner's Simple Garb chest armor 83.0 12.8 5.1 Commoner's Garb chest armor 83.0 12.8 5.1 Noble's Traveling Garb chest armor 83.0 13.3 6.3 Azur's Glintstone Robe chest armor 83.0 13.8 7.1 Traveling Maiden Robe chest armor 83.0 13.4 4.9 Page Garb chest armor 83.0 13.0 5.1 Queen's Robe chest armor 83.0 13.3 5.1 Godskin Noble Robe chest armor 83.0 14.1 6.3 Fell Omen Cloak chest armor 83.0 12.8 5.1 Finger Robe chest armor 82.0 12.8 5.1 Depraved Perfumer Robe chest armor 80.0 12.7 5.0 Mushroom Body chest armor 76.0 13.0 5.1 Astrologer Robe chest armor 76.0 13.0 6.3 Millicent's Tunic chest armor 76.0 13.0 8.3

**The highest vitality-resistant armors in Elden Ring are predominantly light to mid-weight robes, prioritizing health and survivability while keeping mobility manageable. Fia’s Robe and Deathbed Dress lead in vitality resistance, making them ideal for sustained encounters, while Lusat’s Robe and Godskin Noble Robe combine vitality with strong magical defenses for hybrid caster builds. Most top vitality armors maintain moderate weight, allowing players to balance durability, rolling speed, and casting efficiency, catering to tanky, hybrid, or agile playstyles.**

# **📊 14: Calculate "total defense" score**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    (negation_physical + negation_strike + negation_slash + negation_pierce + 
     negation_magic + negation_fire + negation_lightning + negation_holy) AS total_negation,
    (resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) AS total_resistance,
    (negation_physical + negation_strike + negation_slash + negation_pierce + 
     negation_magic + negation_fire + negation_lightning + negation_holy) + 
    ((resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality + resistance_poise) * 0.5) AS defense_score,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY defense_score DESC
LIMIT 30;

armor_name armor_type total_negation total_resistance defense_score weight Fire Prelate Armor chest armor 132.4 315.0 289.9 24.7 Armor of Solitude chest armor 127.3 317.0 285.8 24.0 Verdigris Armor chest armor 131.3 303.0 282.8 25.9 Bull-Goat Armor chest armor 131.7 302.0 282.7 26.5 Omen Armor chest armor 126.9 306.0 279.9 23.1 Gaius's Armor chest armor 126.89999999999999 298.0 275.9 20.1 Lionel's Armor chest armor 127.60000000000001 294.0 274.6 21.2 Armor of Solitude (Altered) chest armor 123.49999999999999 296.0 271.5 23.2 Tree Sentinel Armor chest armor 126.69999999999999 284.0 268.7 18.9 Veteran's Armor chest armor 125.6 283.0 267.1 18.9 Fire Prelate Armor (Altered) chest armor 127.39999999999999 271.0 262.9 23.6 Radahn's Lion Armor chest armor 123.19999999999999 273.0 259.7 17.5 Lionel's Armor (Altered) chest armor 124.39999999999998 268.0 258.4 20.2 Beast Champion Armor chest armor 123.89999999999999 261.0 254.39999999999998 17.5 Banished Knight Armor chest armor 123.7 257.0 252.2 17.5 Tree Sentinel Armor (Altered) chest armor 124.5 239.0 244.0 18.1 Veteran's Armor (Altered) chest armor 122.6 240.0 242.6 17.2 Scaled Armor chest armor 116.8 249.0 241.3 16.0 Malformed Dragon Armor chest armor 118.89999999999999 240.0 238.89999999999998 16.0 Radahn's Lion Armor (Altered) chest armor 120.8 233.0 237.3 16.4 Beast Champion Armor (Altered) chest armor 122.50000000000001 225.0 235.0 16.4 Banished Knight Armor (Altered) chest armor 120.6 220.0 230.6 16.5 Crucible Tree Armor chest armor 115.70000000000002 224.0 227.70000000000002 15.5 Crucible Axe Armor chest armor 115.7 224.0 227.7 15.5 Royal Knight Armor chest armor 117.5 211.0 223.0 15.5 Cleanrot Armor chest armor 111.8 217.0 220.3 15.0 Lusat's Robe chest armor 83.2 269.0 217.7 7.1 Azur's Glintstone Robe chest armor 81.6 272.0 217.6 7.1 Royal Knight Armor (Altered) chest armor 115.89999999999999 198.0 214.89999999999998 15.0 Mushroom Body chest armor 65.6 297.0 214.1 5.1

**The top 30 armors in Elden Ring ranked by overall defense_score are predominantly heavy chest armors, emphasizing both high physical negation and resistance stats. Fire Prelate Armor and Armor of Solitude lead the list, combining strong defenses with moderate weight, while the Bull-Goat and Verdigris Armors offer maximal protection at higher weight. Altered versions of some sets, like Fire Prelate and Armor of Solitude, provide slightly different stat distributions, favoring either total negation or resistance. Notably, lighter magic-focused robes like Lusat's and Azur's appear at the lower end of this list, showing that even with excellent magical resistance, their overall defense score is limited by their light weight and lower physical negation. This demonstrates the trade-off between mobility and total survivability in armor selection.**

# **📊 15: Lightest helm, chest, gauntlets, legs**

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'helm'
ORDER BY weight
LIMIT 3;

armor_name weight negation_physical resistance_poise Commoner's Headband (Altered) 0.7 0.1 1.0 Dancer's Hood 1.0 0.9 1.0 Leather Headband 1.0 1.4 2.0

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'chest armor'
ORDER BY weight
LIMIT 3;

armor_name weight negation_physical resistance_poise Dancer's Dress (Altered) 2.4 2.7 3.0 Goldmask's Rags 2.4 4.2 5.0 Spellblade's Traveling Attire (Altered) 2.5 2.6 5.0

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'gauntlets'
ORDER BY weight
LIMIT 3;

armor_name weight negation_physical resistance_poise Braided Arm Wraps 0.8 1.0 1.0 Brave's Bracer 0.8 0.6 1.0 Gold Bracelets 0.8 1.0 1.0

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'leg armor'
ORDER BY weight
LIMIT 3;

armor_name weight negation_physical resistance_poise Soiled Loincloth 1.5 2.3 2.0 Dancer's Trousers 2.0 2.3 3.0 Ascetic's Ankle Guards 2.0 3.4 6.0

**The lightest armor pieces in Elden Ring by type are clearly designed for mobility rather than defense. For helms, the Commoner's Headband (Altered) at 0.7 weight offers almost no physical negation or poise, while slightly heavier options like the Leather Headband provide a tiny boost. Chest armors such as Dancer's Dress (Altered) and Goldmask’s Rags are extremely light (≈2.4–2.5 weight) with minimal protection. Gauntlets remain under 1 weight, providing very low negation and poise, making them ideal for players prioritizing speed. Finally, leg armor like Soiled Loincloth and Dancer’s Trousers balances light weight with modest poise and physical defense. Overall, these sets are perfect for agile, low-weight builds but sacrifice survivability.**

# **📊 16: Heaviest helm, chest, gauntlets, legs**

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'helm'
ORDER BY weight DESC
LIMIT 3;

armor_name weight negation_physical resistance_poise Greatjar 12.3 8.0 14.0 Pumpkin Helm 12.3 7.0 11.0 Bull-Goat Helm 11.3 7.5 15.0

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'chest armor'
ORDER BY weight DESC
LIMIT 3;

armor_name weight negation_physical resistance_poise Bull-Goat Armor 26.5 20.4 47.0 Verdigris Armor 25.9 20.4 47.0 Fire Prelate Armor 24.7 19.2 45.0

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'gauntlets'
ORDER BY weight DESC
LIMIT 3;

armor_name weight negation_physical resistance_poise Bull-Goat Gauntlets 8.8 5.2 10.0 Verdigris Gauntlets 8.6 5.2 10.0 Fire Prelate Gauntlets 8.2 4.9 10.0

In [0]:
%sql
SELECT armor_name, weight, negation_physical, resistance_poise
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'leg armor'
ORDER BY weight DESC
LIMIT 3;

armor_name weight negation_physical resistance_poise Bull-Goat Greaves 16.4 11.9 28.0 Verdigris Greaves 16.0 11.9 28.0 Fire Prelate Greaves 15.3 11.1 27.0

**The heaviest armor pieces in Elden Ring provide maximum protection at the cost of mobility. For helms, Greatjar and Pumpkin Helm reach over 11 weight, offering high physical negation (7–8) and strong poise (11–15). Chest armors like Bull-Goat Armor, Verdigris Armor, and Fire Prelate Armor top the scale (≈25–26.5 weight) with outstanding negation and poise, making them ideal for tank builds. Gauntlets such as Bull-Goat and Verdigris versions provide solid defensive stats while remaining lighter than chest armor. Leg armor like Bull-Goat Greaves and Verdigris Greaves combine heavy protection with significant poise and physical negation. Overall, these sets are best suited for players prioritizing survivability and tanking over speed and agility.**

# **📊 17: DLC vs base game armor comparison**

In [0]:
%sql
SELECT 
    CASE WHEN a.is_dlc = 1 THEN 'DLC' ELSE 'Base Game' END AS content_type,
    COUNT(*) AS armor_count,
    ROUND(AVG(s.weight), 2) AS avg_weight,
    ROUND(AVG(s.negation_physical), 2) AS avg_physical,
    ROUND(AVG(s.resistance_poise), 2) AS avg_poise
FROM eldenringcatalog.gold.dim_armors a
INNER JOIN eldenringcatalog.gold.fact_armor_stats s ON a.armor_name = s.armor_name
GROUP BY a.is_dlc;

content_type armor_count avg_weight avg_physical avg_poise DLC 145 6.02 6.09 10.82 Base Game 578 5.99 6.12 10.56

**the majority of armors belong to the Base Game (578 pieces), with DLC content adding 145 pieces. On average, DLC armors are slightly heavier (6.02 vs 5.99), have marginally lower physical negation (6.09 vs 6.12), but slightly higher poise (10.82 vs 10.56). This suggests that DLC armors tend to prioritize stability and stagger resistance, offering a subtle defensive advantage while maintaining similar overall weight and physical protection compared to Base Game sets.**

# **📊 18: Best "balanced" armor pieces (good all-around stats)**

In [0]:
%sql
SELECT 
    armor_name,
    armor_type,
    ROUND((negation_physical + negation_strike + negation_slash + negation_pierce) / 4, 2) AS physical_avg,
    ROUND((negation_magic + negation_fire + negation_lightning + negation_holy) / 4, 2) AS elemental_avg,
    ROUND((resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality) / 4, 2) AS resistance_avg,
    resistance_poise,
    ROUND(
        ((negation_physical + negation_strike + negation_slash + negation_pierce) / 4 * 0.4) +
        ((negation_magic + negation_fire + negation_lightning + negation_holy) / 4 * 0.3) +
        ((resistance_immunity + resistance_robustness + resistance_focus + resistance_vitality) / 4 * 0.2) +
        (resistance_poise * 0.1), 2
    ) AS balance_score,
    weight
FROM eldenringcatalog.gold.fact_armor_stats
ORDER BY balance_score DESC
LIMIT 30;

armor_name armor_type physical_avg elemental_avg resistance_avg resistance_poise balance_score weight Fire Prelate Armor chest armor 18.2 14.9 67.5 45.0 29.75 24.7 Armor of Solitude chest armor 18.05 13.77 68.0 45.0 29.45 24.0 Verdigris Armor chest armor 19.13 13.7 64.0 47.0 29.26 25.9 Bull-Goat Armor chest armor 19.3 13.63 63.75 47.0 29.26 26.5 Omen Armor chest armor 17.68 14.05 65.75 43.0 28.74 23.1 Gaius's Armor chest armor 17.85 13.88 65.0 38.0 28.1 20.1 Lionel's Armor chest armor 18.18 13.73 63.5 40.0 28.09 21.2 Armor of Solitude (Altered) chest armor 17.4 13.48 63.25 43.0 27.95 23.2 Tree Sentinel Armor chest armor 17.48 14.2 61.25 39.0 27.4 18.9 Veteran's Armor chest armor 17.98 13.43 61.25 38.0 27.27 18.9 Fire Prelate Armor (Altered) chest armor 17.5 14.35 57.0 43.0 27.01 23.6 Lionel's Armor (Altered) chest armor 17.78 13.33 57.5 38.0 26.41 20.2 Radahn's Lion Armor chest armor 17.38 13.43 59.75 34.0 26.33 17.5 Beast Champion Armor chest armor 17.7 13.28 56.75 34.0 25.81 17.5 Banished Knight Armor chest armor 17.6 13.33 55.75 34.0 25.59 17.5 Tree Sentinel Armor (Altered) chest armor 17.28 13.85 51.0 35.0 24.77 18.1 Veteran's Armor (Altered) chest armor 17.68 12.98 51.25 35.0 24.71 17.2 Scaled Armor chest armor 15.68 13.53 54.0 33.0 24.43 16.0 Malformed Dragon Armor chest armor 16.53 13.2 52.25 31.0 24.12 16.0 Radahn's Lion Armor (Altered) chest armor 16.95 13.25 50.0 33.0 24.06 16.4 Beast Champion Armor (Altered) chest armor 17.45 13.17 47.75 34.0 23.88 16.4 Banished Knight Armor (Altered) chest armor 17.05 13.1 46.75 33.0 23.4 16.5 Crucible Tree Armor chest armor 16.23 12.7 47.75 33.0 23.15 15.5 Crucible Axe Armor chest armor 16.25 12.68 47.75 33.0 23.15 15.5 Royal Knight Armor chest armor 16.18 13.2 46.75 24.0 22.18 15.5 Cleanrot Armor chest armor 15.38 12.58 47.5 27.0 22.12 15.0 Royal Knight Armor (Altered) chest armor 15.98 13.0 42.5 28.0 21.59 15.0 Young Lion's Armor chest armor 16.2 12.35 40.75 31.0 21.44 15.0 Scaled Armor (Altered) chest armor 15.0 12.83 41.75 30.0 21.2 15.0 Crucible Axe Armor (Altered) chest armor 15.55 12.4 40.25 31.0 21.09 14.9

**Based on the balance score analysis, Fire Prelate Armor, Armor of Solitude, and Verdigris Armor rank highest, excelling in a well-rounded combination of physical protection, elemental defense, general resistances, and poise. Heavy chest armors dominate the top positions, reflecting a trend where higher weight correlates with better overall defensive balance. The scoring system highlights armors that provide consistent protection across all damage types while maintaining significant poise, making them optimal choices for players seeking survivability in varied combat scenarios.**

# **📊 19: Best light armor set (total weight < 20)**

In [0]:
%sql
SELECT 
    armor_name,
    negation_physical,
    weight,
    ROUND(negation_physical / weight, 2) AS efficiency
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'helm' AND weight <= 5
ORDER BY efficiency DESC
LIMIT 3;

armor_name negation_physical weight efficiency Circlet of Light 2.8 1.0 2.8 Aristocrat Headband 1.9 1.2 1.58 Leather Headband 1.4 1.0 1.4

In [0]:
%sql
SELECT 
    armor_name,
    negation_physical,
    weight,
    ROUND(negation_physical / weight, 2) AS efficiency
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'chest armor' AND weight <= 5
ORDER BY efficiency DESC
LIMIT 3;

armor_name negation_physical weight efficiency Ascetic's Loincloth 6.1 3.2 1.91 Braided Cord Robe 6.3 3.4 1.85 Goldmask's Rags 4.2 2.4 1.75

In [0]:
%sql
SELECT 
    armor_name,
    negation_physical,
    weight,
    ROUND(negation_physical / weight, 2) AS efficiency
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'gauntlets' AND weight <= 5
ORDER BY efficiency DESC
LIMIT 3;

armor_name negation_physical weight efficiency Ascetic's Wrist Guards 1.5 1.1 1.36 Braided Arm Wraps 1.0 0.8 1.25 Gold Bracelets 1.0 0.8 1.25

In [0]:
%sql
SELECT 
    armor_name,
    negation_physical,
    weight,
    ROUND(negation_physical / weight, 2) AS efficiency
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'leg armor' AND weight <= 5
ORDER BY efficiency DESC
LIMIT 3;

armor_name negation_physical weight efficiency Ascetic's Ankle Guards 3.4 2.0 1.7 Old Aristocrat Shoes 3.4 2.0 1.7 Soiled Loincloth 2.3 1.5 1.53

**For light armor (weight ≤ 5), certain pieces stand out for their exceptional physical defense efficiency. Among helms, the Circlet of Light, Aristocrat Headband, and Leather Headband offer the best protection relative to their weight. In chest armor, Ascetic's Loincloth, Braided Cord Robe, and Goldmask's Rags provide a strong balance of lightness and defense. For gauntlets, Ascetic's Wrist Guards and Braided Arm Wraps deliver solid protection without slowing the player down. Lastly, light leg armor such as Ascetic's Ankle Guards and Old Aristocrat Shoes achieves high defense per weight, making these pieces ideal for agile characters seeking to maintain mobility while staying well-protected.**

# **📊 20: Best heavy armor set (max protection)**

In [0]:
%sql
SELECT armor_name, negation_physical, resistance_poise, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'helm'
ORDER BY negation_physical DESC
LIMIT 3;

armor_name negation_physical resistance_poise weight Greatjar 8.0 14.0 12.3 Verdigris Helm 7.5 15.0 11.1 Bull-Goat Helm 7.5 15.0 11.3

In [0]:
%sql
SELECT armor_name, negation_physical, resistance_poise, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'chest armor'
ORDER BY negation_physical DESC
LIMIT 3;

armor_name negation_physical resistance_poise weight Verdigris Armor 20.4 47.0 25.9 Bull-Goat Armor 20.4 47.0 26.5 Fire Prelate Armor 19.2 45.0 24.7

In [0]:
%sql
SELECT armor_name, negation_physical, resistance_poise, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'gauntlets'
ORDER BY negation_physical DESC
LIMIT 3;

armor_name negation_physical resistance_poise weight Verdigris Gauntlets 5.2 10.0 8.6 Bull-Goat Gauntlets 5.2 10.0 8.8 Fire Prelate Gauntlets 4.9 10.0 8.2

In [0]:
%sql
SELECT armor_name, negation_physical, resistance_poise, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'leg armor'
ORDER BY negation_physical DESC
LIMIT 3;

armor_name negation_physical resistance_poise weight Verdigris Greaves 11.9 28.0 16.0 Bull-Goat Greaves 11.9 28.0 16.4 Fire Prelate Greaves 11.1 27.0 15.3

**For heavy-hitting armor pieces that maximize physical defense, certain sets dominate each category. Among helms, Greatjar, Verdigris Helm, and Bull-Goat Helm offer the highest physical negation, paired with strong poise but substantial weight. In chest armor, Verdigris Armor, Bull-Goat Armor, and Fire Prelate Armor lead with top-tier physical protection and high poise, making them ideal for tanky builds. Gauntlets like Verdigris, Bull-Goat, and Fire Prelate provide solid hand protection without being overly cumbersome. For leg armor, Verdigris Greaves, Bull-Goat Greaves, and Fire Prelate Greaves combine excellent physical defense with notable poise, completing some of the most formidable heavy armor sets in the game.**

# **📊 21: Armor for specific boss types (fire, magic, lightning, holy)**

In [0]:
%sql
-- FIRE RESISTANCE ARMOR (for fire bosses)
WITH max_fire AS (
    SELECT armor_type, MAX(negation_fire) AS max_fire
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)
SELECT a.armor_name, a.armor_type, a.negation_fire
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_fire m ON a.armor_type = m.armor_type AND a.negation_fire = m.max_fire
ORDER BY a.armor_type

armor_name armor_type negation_fire Fire Prelate Armor chest armor 19.8 Fire Prelate Gauntlets gauntlets 5.0 Divine Beast Head helm 7.2 Fire Prelate Helm helm 7.2 Fire Prelate Greaves leg armor 11.5

In [0]:
%sql
-- MAGIC RESISTANCE ARMOR (for sorcery bosses)
WITH max_magic AS (
    SELECT armor_type, MAX(negation_magic) AS max_magic
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)
SELECT a.armor_name, a.armor_type, a.negation_magic
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_magic m ON a.armor_type = m.armor_type AND a.negation_magic = m.max_magic
ORDER BY a.armor_type;

armor_name armor_type negation_magic Azur's Glintstone Robe chest armor 15.4 Lusat's Robe chest armor 15.4 Preceptor's Gloves gauntlets 3.6 Nox Mirrorhelm helm 6.7 Preceptor's Trousers leg armor 8.3

In [0]:
%sql
-- LIGHTNING RESISTANCE ARMOR (for lightning bosses)
WITH max_lightning AS (
    SELECT armor_type, MAX(negation_lightning) AS max_lightning
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)
SELECT a.armor_name, a.armor_type, a.negation_lightning
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_lightning m ON a.armor_type = m.armor_type AND a.negation_lightning = m.max_lightning
ORDER BY a.armor_type;

armor_name armor_type negation_lightning Omen Armor chest armor 14.9 Bull-Goat Armor chest armor 14.9 Bull-Goat Gauntlets gauntlets 3.7 Omen Gauntlets gauntlets 3.7 Greatjar helm 5.5 Greathood helm 5.5 Bull-Goat Greaves leg armor 8.5 Omen Greaves leg armor 8.5

**In Elden Ring, certain armor sets stand out for elemental resistances, making them ideal for specific boss encounters. For fire-heavy enemies, the Fire Prelate set excels, offering top-tier fire negation across chest armor, helm, gauntlets, and greaves, with the Divine Beast Head helm also providing strong fire protection. Against magic-based foes, chest armors like Azur’s Glintstone Robe and Lusat’s Robe deliver the highest magic defense, complemented by Preceptor’s Gloves, Nox Mirrorhelm, and Preceptor’s Trousers for gauntlets, helm, and legs. For lightning challenges, the Bull-Goat and Omen sets dominate, providing exceptional lightning resistance in all armor slots. Choosing these sets ensures a balance of elemental protection, poise, and overall defense for tough battles.**

# **📊 22: Best poise armor set (maximum hyper-armor)**


In [0]:
%sql
SELECT armor_name, resistance_poise, negation_physical, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'helm'
ORDER BY resistance_poise DESC
LIMIT 3;

armor_name resistance_poise negation_physical weight Verdigris Helm 15.0 7.5 11.1 Bull-Goat Helm 15.0 7.5 11.3 Helm of Solitude 14.0 6.8 10.3

In [0]:
%sql
SELECT armor_name, resistance_poise, negation_physical, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'chest armor'
ORDER BY resistance_poise DESC
LIMIT 3;

armor_name resistance_poise negation_physical weight Verdigris Armor 47.0 20.4 25.9 Bull-Goat Armor 47.0 20.4 26.5 Armor of Solitude 45.0 18.7 24.0

In [0]:
%sql
SELECT armor_name, resistance_poise, negation_physical, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'gauntlets'
ORDER BY resistance_poise DESC
LIMIT 3;

armor_name resistance_poise negation_physical weight Verdigris Gauntlets 10.0 5.2 8.6 Gauntlets of Solitude 10.0 4.7 8.0 Bull-Goat Gauntlets 10.0 5.2 8.8

In [0]:
%sql
SELECT armor_name, resistance_poise, negation_physical, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'leg armor'
ORDER BY resistance_poise DESC
LIMIT 3;

armor_name resistance_poise negation_physical weight Verdigris Greaves 28.0 11.9 16.0 Bull-Goat Greaves 28.0 11.9 16.4 Greaves of Solitude 27.0 10.8 14.8

**In Elden Ring, armor with high poise is essential for withstanding stagger and maintaining stability during combat. The top poise helms include Verdigris Helm and Bull-Goat Helm, both offering 15 poise along with strong physical negation. For chest armor, Verdigris Armor and Bull-Goat Armor lead with 47 poise, making them ideal for tanking hits, while Armor of Solitude provides 45 poise. In the gauntlets category, Verdigris Gauntlets, Gauntlets of Solitude, and Bull-Goat Gauntlets all reach 10 poise. For leg armor, Verdigris Greaves and Bull-Goat Greaves top the list with 28 poise, followed closely by Greaves of Solitude at 27. These sets combine high poise with substantial physical negation, offering excellent defensive stability for melee-focused builds.**

# **📊 23: Create armor weight categories**

In [0]:
%sql
SELECT 
    armor_type,
    CASE 
        WHEN weight <= 3 THEN 'Light'
        WHEN weight <= 7 THEN 'Medium'
        WHEN weight <= 12 THEN 'Heavy'
        ELSE 'Very Heavy'
    END AS weight_class,
    COUNT(*) AS piece_count,
    ROUND(AVG(negation_physical), 2) AS avg_physical,
    ROUND(AVG(resistance_poise), 2) AS avg_poise
FROM eldenringcatalog.gold.fact_armor_stats
GROUP BY armor_type, 
    CASE 
        WHEN weight <= 3 THEN 'Light'
        WHEN weight <= 7 THEN 'Medium'
        WHEN weight <= 12 THEN 'Heavy'
        ELSE 'Very Heavy'
    END
ORDER BY armor_type, weight_class;

armor_type weight_class piece_count avg_physical avg_poise chest armor Heavy 120 11.17 18.76 chest armor Light 3 3.17 4.33 chest armor Medium 87 5.46 8.8 chest armor Very Heavy 41 16.78 33.41 gauntlets Heavy 6 4.83 9.67 gauntlets Light 62 1.76 2.56 gauntlets Medium 55 3.43 5.27 helm Heavy 26 6.31 11.42 helm Light 65 1.88 2.94 helm Medium 119 3.91 6.63 helm Very Heavy 2 7.5 12.5 leg armor Heavy 33 8.44 16.0 leg armor Light 28 2.82 4.36 leg armor Medium 70 5.57 9.34 leg armor Very Heavy 6 11.07 26.5

**armor table shows a clear relationship between weight classes and defensive stats. Light armor pieces are the most numerous for helms and gauntlets, offering low physical negation and poise, making them ideal for mobility-focused builds. Medium armor provides moderate protection and poise across all types, balancing defense and agility. Heavy armor excels in poise and physical negation, with chest and leg armor averaging over 16–18 poise and 11–12 physical negation. The very heavy class, though rare, delivers the highest defensive stats, with chest armor reaching an average of 33.41 poise and 16.78 physical negation, ideal for tank builds at the cost of mobility. This classification helps players choose armor tailored to their playstyle, whether nimble or heavily armored.**

# **📊 24: Armor recommendations by playstyle**

## STRENGTH BUILD (Heavy armor, high poise)

In [0]:
%sql
SELECT armor_name, resistance_poise, negation_physical, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'helm'
ORDER BY resistance_poise DESC
LIMIT 1;

armor_name resistance_poise negation_physical weight Verdigris Helm 15.0 7.5 11.1

In [0]:
%sql
SELECT armor_name, resistance_poise, negation_physical, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'chest armor'
ORDER BY resistance_poise DESC
LIMIT 1;

armor_name resistance_poise negation_physical weight Verdigris Armor 47.0 20.4 25.9

In [0]:
%sql
SELECT armor_name, resistance_poise, negation_physical, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'gauntlets'
ORDER BY resistance_poise DESC
LIMIT 1;

armor_name resistance_poise negation_physical weight Verdigris Gauntlets 10.0 5.2 8.6

In [0]:
%sql
SELECT armor_name, resistance_poise, negation_physical, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'leg armor'
ORDER BY resistance_poise DESC
LIMIT 1;

armor_name resistance_poise negation_physical weight Verdigris Greaves 28.0 11.9 16.0

**For a Strength-focused build in Elden Ring, prioritizing high poise and heavy armor, the top choices for each armor slot are: Verdigris Helm for the head, Verdigris Armor for the chest, Verdigris Gauntlets for the hands, and Verdigris Greaves for the legs. These pieces provide exceptional poise values—15 for the helm, 47 for the chest, 10 for the gauntlets, and 28 for the greaves—while also offering solid physical negation (ranging from 5.2 to 20.4) and heavy weight, making them ideal for tanky, melee-focused builds that can withstand hits while dealing powerful attacks.**

## DEXTERITY BUILD (Light armor, mobility)

In [0]:
%sql
SELECT armor_name, negation_physical, weight, 
       ROUND(negation_physical / weight, 2) AS efficiency
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'helm' AND weight <= 5
ORDER BY efficiency DESC
LIMIT 1;

armor_name negation_physical weight efficiency Circlet of Light 2.8 1.0 2.8

In [0]:
%sql
SELECT armor_name, negation_physical, weight, 
       ROUND(negation_physical / weight, 2) AS efficiency
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'chest armor' AND weight <= 5
ORDER BY efficiency DESC
LIMIT 1;

armor_name negation_physical weight efficiency Ascetic's Loincloth 6.1 3.2 1.91

In [0]:
%sql
SELECT armor_name, negation_physical, weight, 
       ROUND(negation_physical / weight, 2) AS efficiency
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'gauntlets' AND weight <= 5
ORDER BY efficiency DESC
LIMIT 1;

armor_name negation_physical weight efficiency Ascetic's Wrist Guards 1.5 1.1 1.36

In [0]:
%sql
SELECT armor_name, negation_physical, weight, 
       ROUND(negation_physical / weight, 2) AS efficiency
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'leg armor' AND weight <= 5
ORDER BY efficiency DESC
LIMIT 1;

armor_name negation_physical weight efficiency Ascetic's Ankle Guards 3.4 2.0 1.7

**For a Dexterity-focused build that prioritizes mobility and efficiency in Elden Ring, the best lightweight armor pieces are: Circlet of Light for the helm, Ascetic's Loincloth for the chest, Ascetic's Wrist Guards for the gauntlets, and Ascetic's Ankle Guards for the legs. These selections maximize physical negation relative to weight, offering high efficiency while keeping the overall armor load low, making them perfect for fast, agile combat and quick dodging.**

## MAGIC BUILD (High focus resistance)

In [0]:
%sql
SELECT armor_name, resistance_focus, negation_magic, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'helm'
ORDER BY resistance_focus DESC
LIMIT 1;

armor_name resistance_focus negation_magic weight Greathood 47.0 5.5 5.1

In [0]:
%sql
SELECT armor_name, resistance_focus, negation_magic, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'chest armor'
ORDER BY resistance_focus DESC
LIMIT 1;

armor_name resistance_focus negation_magic weight Corhyn's Robe 107.0 13.3 6.3

In [0]:
%sql
SELECT armor_name, resistance_focus, negation_magic, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'gauntlets'
ORDER BY resistance_focus DESC
LIMIT 1;


armor_name resistance_focus negation_magic weight Braided Arm Wraps 34.0 3.1 0.8

In [0]:
%sql
SELECT armor_name, resistance_focus, negation_magic, weight
FROM eldenringcatalog.gold.fact_armor_stats
WHERE armor_type = 'leg armor'
ORDER BY resistance_focus DESC
LIMIT 1;

armor_name resistance_focus negation_magic weight Soiled Loincloth 62.0 7.1 1.5

**For a Magic-focused build in Elden Ring, which emphasizes high focus resistance and protection against sorceries, the top armor pieces are: Greathood for the helm, Corhyn's Robe for the chest, Braided Arm Wraps for the gauntlets, and Soiled Loincloth for the legs. These selections maximize resistance to magic attacks while keeping weight manageable, providing optimal defense for spellcasters and sorcery-heavy encounters.**

## FIRE RESISTANCE BUILD (Against fire bosses)

In [0]:
%sql
WITH max_fire AS (
    SELECT armor_type, MAX(negation_fire) AS max_fire
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_fire
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_fire m 
    ON a.armor_type = m.armor_type AND a.negation_fire = m.max_fire
WHERE a.armor_type = 'helm';

armor_name armor_type negation_fire Divine Beast Head helm 7.2 Fire Prelate Helm helm 7.2

In [0]:
%sql
WITH max_fire AS (
    SELECT armor_type, MAX(negation_fire) AS max_fire
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_fire
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_fire m 
    ON a.armor_type = m.armor_type AND a.negation_fire = m.max_fire
WHERE a.armor_type = 'chest armor';

armor_name armor_type negation_fire Fire Prelate Armor chest armor 19.8

In [0]:
%sql
WITH max_fire AS (
    SELECT armor_type, MAX(negation_fire) AS max_fire
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_fire
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_fire m 
    ON a.armor_type = m.armor_type AND a.negation_fire = m.max_fire
WHERE a.armor_type = 'gauntlets';

armor_name armor_type negation_fire Fire Prelate Gauntlets gauntlets 5.0

In [0]:
%sql
WITH max_fire AS (
    SELECT armor_type, MAX(negation_fire) AS max_fire
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_fire
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_fire m 
    ON a.armor_type = m.armor_type AND a.negation_fire = m.max_fire
WHERE a.armor_type = 'leg armor';

armor_name armor_type negation_fire Fire Prelate Greaves leg armor 11.5

**For fire resistance, the optimal armor pieces are Divine Beast Head and Fire Prelate Helm for helms, Fire Prelate Armor for chest, Fire Prelate Gauntlets for gauntlets, and Fire Prelate Greaves for leg armor. These selections provide the maximum fire negation in their respective categories, making them the most effective for encounters with fire-based bosses.**

## MAGIC BUILD (High focus resistance) - Best helm

In [0]:
%sql
WITH max_magic AS (
    SELECT armor_type, MAX(negation_magic) AS max_magic
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)
SELECT a.armor_name, a.armor_type, a.negation_magic
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_magic m 
    ON a.armor_type = m.armor_type AND a.negation_magic = m.max_magic
WHERE a.armor_type = 'helm';

armor_name armor_type negation_magic Nox Mirrorhelm helm 6.7

In [0]:
%sql
WITH max_magic AS (
    SELECT armor_type, MAX(negation_magic) AS max_magic
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_magic
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_magic m 
    ON a.armor_type = m.armor_type AND a.negation_magic = m.max_magic
WHERE a.armor_type = 'chest armor';

armor_name armor_type negation_magic Azur's Glintstone Robe chest armor 15.4 Lusat's Robe chest armor 15.4

In [0]:
%sql
WITH max_magic AS (
    SELECT armor_type, MAX(negation_magic) AS max_magic
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_magic
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_magic m 
    ON a.armor_type = m.armor_type AND a.negation_magic = m.max_magic
WHERE a.armor_type = 'gauntlets';

armor_name armor_type negation_magic Preceptor's Gloves gauntlets 3.6

In [0]:
%sql
WITH max_magic AS (
    SELECT armor_type, MAX(negation_magic) AS max_magic
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_magic
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_magic m 
    ON a.armor_type = m.armor_type AND a.negation_magic = m.max_magic
WHERE a.armor_type = 'leg armor';

armor_name armor_type negation_magic Preceptor's Trousers leg armor 8.3

**For magic resistance, the top armor pieces are Nox Mirrorhelm for helms, Azur's Glintstone Robe and Lusat's Robe for chest armor, Preceptor's Gloves for gauntlets, and Preceptor's Trousers for leg armor. These selections maximize magic negation in each category, providing the most effective protection against sorcery-based attacks and magic-focused bosses.**

## LIGHTNING RESISTANCE BUILD (Against lightning bosses)

In [0]:
%sql
WITH max_lightning AS (
    SELECT armor_type, MAX(negation_lightning) AS max_lightning
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)
SELECT a.armor_name, a.armor_type, a.negation_lightning
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_lightning m 
    ON a.armor_type = m.armor_type AND a.negation_lightning = m.max_lightning
WHERE a.armor_type = 'helm';

armor_name armor_type negation_lightning Greatjar helm 5.5 Greathood helm 5.5

In [0]:
%sql
WITH max_lightning AS (
    SELECT armor_type, MAX(negation_lightning) AS max_lightning
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_lightning
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_lightning m 
    ON a.armor_type = m.armor_type AND a.negation_lightning = m.max_lightning
WHERE a.armor_type = 'chest armor';

armor_name armor_type negation_lightning Bull-Goat Armor chest armor 14.9 Omen Armor chest armor 14.9

In [0]:
%sql
WITH max_lightning AS (
    SELECT armor_type, MAX(negation_lightning) AS max_lightning
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_lightning
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_lightning m 
    ON a.armor_type = m.armor_type AND a.negation_lightning = m.max_lightning
WHERE a.armor_type = 'gauntlets';

armor_name armor_type negation_lightning Bull-Goat Gauntlets gauntlets 3.7 Omen Gauntlets gauntlets 3.7

In [0]:
%sql
WITH max_lightning AS (
    SELECT armor_type, MAX(negation_lightning) AS max_lightning
    FROM eldenringcatalog.gold.fact_armor_stats
    GROUP BY armor_type
)

SELECT a.armor_name, a.armor_type, a.negation_lightning
FROM eldenringcatalog.gold.fact_armor_stats a
INNER JOIN max_lightning m 
    ON a.armor_type = m.armor_type AND a.negation_lightning = m.max_lightning
WHERE a.armor_type = 'leg armor';

armor_name armor_type negation_lightning Bull-Goat Greaves leg armor 8.5 Omen Greaves leg armor 8.5

**For lightning resistance, the top armor pieces are Greatjar and Greathood for helms, Bull-Goat Armor and Omen Armor for chest armor, Bull-Goat Gauntlets and Omen Gauntlets for gauntlets, and Bull-Goat Greaves and Omen Greaves for leg armor. These armors provide the maximum lightning negation in their respective categories, making them ideal for encounters against lightning-based bosses or areas with high electrical damage.**